packages/react/src/analytics/integrations/AnalyticsApi/AnalyticsApi.js

```diff
@@ -0,0 +1,101 @@
+import { integrations, utils } from '@farfetch/blackout-core/analytics';
+import client from '@farfetch/blackout-core/helpers/client';
+
+export default class AnalyticsAPI extends integrations.Integration {
+  static [utils.CONSENT_CATEGORIES_PROPERTY] =
+    utils.DefaultConsentKeys.MARKETING;
+
+  /**
+   * Creates an instance of Analytics Api integration.
+   *
+   * @param {object} options               - User configured options.
+   * @param {object} loadData              - Analytics's load event data.
+   * @param strippedDownAnalytics - Analytics stripped down instance.
+   */
+  constructor(options, loadData, strippedDownAnalytics) {
+    super(options, loadData, strippedDownAnalytics);
+    this.debugMode = utils.getCookie('analyticsAPIDebug');
+  }
+
+  /**
+   * Creates and returns the instance for the Analytics API integration.
+   *
+   * @param {object} options - Integration options.
+   * @param {object} loadData - Analytics' load event data.
+   * @param {object} analytics - Analytics stripped down instance.
+   * @param strippedAnalytics
+   * @returns {AnalyticsAPI} The instance created.
+   */
+  static createInstance(options, loadData, strippedAnalytics) {
+    return new this(options, loadData, strippedAnalytics);
+  }
+
+  /**
+   * Track method that will call Analytics API with the necessary data per event.
+   * If there's more than one pageType per event, it will call Analytics API with each page type at a time.
+   *
+   * @async
+   *
+   * @param {object} data - Track event data.
+   *
+   * @returns {Promise} Promise that will resolve when the method finishes.
+   */
+  async track(data) {
+    if (
+      this.options.conversionsAPI.eventsList &&
+      this.options.conversionsAPI.eventsList.includes(data.event)
+    ) {
+      return await this.callAPI({
+        data,
+        options: this.getOptions(),
+      });
+    }
+    if (this.debugMode) {
```

Comment:
this condition should be inside the previous if, otherwise all tracks that will not reach the analytics API will also be logged.